exonum/src/helpers/fabric/mod.rs

```diff
@@ -191,6 +224,11 @@ impl Context {
         }
     }
 
+    /// Gets the variable from the context.
+    pub fn get2<'de, T: Deserialize<'de>>(&self, key: Key<T>) -> Result<T, Box<Error>> {
```

Comment:
Obviously, this will be named `get` in real implementation. Note the crucial property: when the user supplies some key, the don't need to specify `T` explicitly, and they can't specify the wrong T even if they want to.